demos/projects/ESPRESSIF/aziotkit/main/azure_iot_freertos_esp32_main.c

```diff
@@ -0,0 +1,405 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdarg.h>
+#include <string.h>
+
+#include "sdkconfig.h"
+#include "esp_event.h"
+#include "esp_wifi.h"
+#include "esp_wifi_default.h"
+#include "esp_err.h"
+#include "esp_netif.h"
+#include "esp_sntp.h"
+#include "esp_log.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "freertos/semphr.h"
+#include "nvs_flash.h"
+
+/* Azure Provisioning/IoT Hub library includes */

```

Comment:
Adding the SDK headers as this module will need them to implement the interface to talk to the core module of sample_azure_iot_pnp (defined by `sample_azure_iot_pnp.h`).

Reply:
Missing here: include header `sample_azure_iot_pnp.h`